llvm/test/Transforms/LoopVectorize/simplify-evl-phi.ll

```diff
@@ -0,0 +1,68 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
```

Comment:
For consistency with other related tests, would be good to call it `vector-loop-backedge-elimination-with-evl.ll` or something like tat